code/modules/clothing/spacesuits/rig_modules/rig_modules.dm

```diff
@@ -119,4 +119,31 @@
 	say_to_wearer("Plasma seal disengaged.")
 	R.clothing_flags &= ~PLASMAGUARD
 	if(R.H)
-		R.H.clothing_flags &= ~PLASMAGUARD
\ No newline at end of file
+		R.H.clothing_flags &= ~PLASMAGUARD
+
+/obj/item/rig_module/muscle_tissue
+	name = "artificial muscle tissue"
+	desc = "A flexible tissue with a number of sensors stretched between its surface and interior of the suit. When these sensors detected an impact, the artificial muscle reacts instantaneously, contracting and diffusing the damage."
+
+/obj/item/rig_module/muscle_tissue/activate(var/mob/user,var/obj/item/clothing/suit/space/rig/R)
+	..()
+	if(R.cell.use(5000))
```

Comment:
I've always thought the way the rig module system works is weird

Turning your hulk on and off 10 times costs 50,000 charge but leaving hulk on all shift costs 5,000

Reply:
Maybe a processing object is better? It's kinda weird how we cannot disable rig modules individually, too.

Reply:
Yes, I would definitely prefer them as processing